shared/version/api.go

```diff
@@ -377,6 +377,7 @@ var APIExtensions = []string{
 	"instances_rebuild",
 	"numa_cpu_placement",
 	"custom_volume_iso",
+	"network_allocations",
```

Comment:
Please ensure the api extension change is its own commit accompanied by the associated api extension documentation update, e.g. https://github.com/canonical/lxd/commits/main/doc/api-extensions.md